ajax/libs/algebra.js/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "algebra.js",
+  "version": "0.2.6",
```

Comment:
Could you please help me pull the commits with sparseCheckout? I tried but it doesn't seem to work.